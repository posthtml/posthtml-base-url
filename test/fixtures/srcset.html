<img src="image1.jpg" srcset="image1-HD.jpg 2x,image1-phone.jpg 100w">

<img src="https://example.com/image2.jpg" srcset="https://example.com/image2-HD.jpg 2x, https://example.com/image2-phone.jpg 100w">

<picture>
  <source media="(max-width: 799px)" srcset="elva-480w-close-portrait.jpg">
  <source media="(min-width: 800px)" srcset="elva-800w.jpg">
  <img src="elva-800w.jpg" alt="...">
</picture>
