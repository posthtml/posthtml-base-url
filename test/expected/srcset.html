<img src="https://example.com/image1.jpg" srcset="https://example.com/image1-HD.jpg 2x, https://example.com/image1-phone.jpg 100w">

<img src="https://example.com/image2.jpg" srcset="https://example.com/image2-HD.jpg 2x, https://example.com/image2-phone.jpg 100w">

<picture>
  <source media="(max-width: 799px)" srcset="https://example.com/elva-480w-close-portrait.jpg">
  <source media="(min-width: 800px)" srcset="https://example.com/elva-800w.jpg">
  <img src="https://example.com/elva-800w.jpg" alt="...">
</picture>
